src/Impostor.Api.Innersloth.Generator/SourceGenerator.cs

```diff
@@ -75,6 +75,7 @@ public void Initialize(IncrementalGeneratorInitializationContext context)
             enumGenerator.Generate("ColorType", "Impostor.Api.Innersloth.Customization");
 
             enumGenerator.Generate("DisconnectReason", sourceName: "DisconnectReasons");
+            enumGenerator.Generate("SanctionReason", sourceName: "SanctionReasons");
```

Comment:
What's the reason for renaming it to `SanctionReason`? Both Impostor and base game are inconsistent in naming of enums, so I think we should just stick with base game's `SanctionReasons`.